Move generating of MetricRollups to one method in specs

lpichler opened the pull request:
@miq-bot assign @chrisarcand 

@miq-bot add_label test, chargeback, refactoring, technical_debt

chrisarcand commented on spec/support/chargeback_helper.rb:
```diff
@@ -10,6 +10,18 @@ def set_tier_param_for(metric, param, value, num_of_tier = 0)
         resource.metric_rollups.sum(&metric) / hours_in_interval
       end
+
+      def add_metric_rollups_for(resources, range, step, custom_params, trait = :with_data)
```
1) Is the `trait` parameter used at all? If everything uses the default, maybe just don't even bother passing it in until it's actually needed.

2) Let's make this method handle singles vs multiples. Do `Array(resources).each do |resource| ...` and update all the callers that pass an array of one resource just to pass the resource. 

3) `custom_params` doesn't have any context; can you rename to something like `metric_rollup_params` or something else that says 'these params are what is passed to create a metric rollup'?

lpichler replied:
thanks @chrisarcand, updated 👍 
1. Ok, will add when it will be needed.
2. 👍 
3. 👍

chrisarcand commented:
@lpichler Looks like you missed one:

```
     ArgumentError:
       wrong number of arguments (given 5, expected 4)
     # ./spec/support/chargeback_helper.rb:14:in `add_metric_rollups_for'
     # ./spec/models/chargeback_container_project_spec.rb:170:in `block (3 levels) in <top (required)>'
```

lpichler commented:
> @lpichler Looks like you missed one:

@chrisarcand  Yes, Fixed.
1. I added `trait` parameter back. I realized that I have one [case here for it.](https://github.com/ManageIQ/manageiq/pull/13858/files#diff-4a20ba9265ccf9f0cef951e11c6205a0L189)
